R/RunDiagnostics.R

```diff
@@ -238,25 +238,7 @@ executeDiagnostics <- function(cohortDefinitionSet,
                                seed = 64374,
                                seedArgs = NULL,
                                sampleIdentifierExpression = "cohortId * 1000 + seed") {
-  # collect arguments that were passed to cohort diagnostics at initiation
-  callingArgs <- formals(executeDiagnostics)
-  callingArgsJson <-
```

Comment:
let's just leave this in, since it's not related to the goal of this PR

Reply:
agreed.